uvloop/__init__.py

```diff
@@ -1,6 +1,6 @@
 import asyncio as __asyncio
 import typing as _typing
-import sys as _sys
```

Comment:
mypy doesn't recognise `_sys.version_info` in comparisons it needs the real `sys.version_info`